runners/twister2/build.gradle

```diff
@@ -36,7 +36,22 @@ dependencies {
     compile (project(path: ":runners:core-java")){
         exclude group: 'com.esotericsoftware.kryo', module: 'kryo'
     }
-    compile project(":runners:java-fn-execution")
+    compile project(path: ":model:pipeline", configuration: "shadow")
+    compile project(path: ":runners:core-construction-java")
+    compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile library.java.jackson_annotations
+    compile library.java.joda_time
+    compile library.java.vendored_grpc_1_26_0
+    compile "org.checkerframework:checker-qual:2.0.0"
+    compile "org.mortbay.jetty:jetty-util:6.1.26"
+    compile library.java.slf4j_api
```

Comment:
Move this up with the other `library.` definitions.

Reply:
done, thanks!